src/main/java/Alex/Command/MarkCommand.java

```diff
@@ -0,0 +1,33 @@
+package Alex.Command;
+import Alex.Exceptions.AlexException;
+import Alex.Storage.Storage;
+import Alex.Task.Task;
+import Alex.Task.TaskList;
+import Alex.Ui.Ui;
+
+public class MarkCommand extends CommandBase {
+    private int index;
+    private boolean markAsDone;
```

Comment:
I think it might be better to follow the coding standard and use something like isDone since this is a boolean?